### PR TITLE
fix(gui): Add Status View for Contracts

### PR DESCRIPTION
**Pull request by themylogin**
Ticket: #26213

### CODE REVIEW

File: gui/templates/system/system_info.html [hunk 1]
```diff
@@ -102,6 +102,10 @@ <h1>{% trans "System Information" %}</h1>
         <th>{% trans "System Product" %}</th>
         <td class="descr">{{ local.system_product }}</td>
     </tr>
+    <tr>
+        <th>{% trans "License" %}</th>
+        <td class="descr">{{ local.license }}</td>
+    </tr>
```
Comment (william-gr) [inline gui/templates/system/system_info.html]: What if there is no license? Should we encapsulate that block within {% if license %}?

Comment (themylogin) [reply gui/templates/system/system_info.html]: Or should we display red text "No license"?

File: src/middlewared/middlewared/plugins/system.py [hunk 2]
```diff
@@ -64,6 +69,22 @@ def ready(self):
...
+                    _(ContractType(license.contract_type).name.title()),
+                    license.contract_end.strftime("%x"),
+                    days,
+                    ungettext('day', 'days', days),
+                ))
+            else:
+                license = _('%1s contract, expired at %2s' % (
+                    _(ContractType(license.contract_type).name.title()),
+                    license.contract_end.strftime("%x"),
+                ))
```
Comment (william-gr) [inline src/middlewared/middlewared/plugins/system.py]: I am not a fan of providing a human readable output for `license`. Any way we can provide a dict with the desired data and format that on the UI side?

File: gui/templates/system/system_info.html (hunk 1 above)
Comment (william-gr) [reply gui/templates/system/system_info.html]: License is very rare on FreeNAS. Perhaps show "No license" for TrueNAS and hide it from FreeNAS?

Comment (themylogin) [reply gui/templates/system/system_info.html]: OK. It's already hidden from FreeNAS btw (see full context)

**Pull request merged**